### ISSUE TITLE
field like `company_id` always export to companyId on iOS

### ISSUE THREAD

**Issue by Jerret321** [Ahmed-Ali/JSONExport#49]
I think the same problem will accour on Android. I think sometimes the model fields like 'aaa_bbb_ccc' are used in db model. If the field export camel field like aaaBbbCcc, it's so serious.

**Comment by Ahmed-Ali** [comment]
this change happens only for the fieldName not the json key, right?
in such case, this is not a bug. It meant to be like that, because in Objective-C, Swift, and Java the convention is to use camel case instead of underscores.

**Comment by Jerret321** [comment]
Yes, that's ok. But I think not every app design the field name with underscores. So can you give more choice for users?

**Comment by Ahmed-Ali** [comment]
I agree there should be an option for that.
Will consider it as soon as possible!

**Comment by Jerret321** [comment]
Thanks too much

**Comment by mhtranbn** [comment]
me need too, many thanks for supports.

**Comment by hjqiaho** [comment]
dele func underscoresToCamelCaseForString
